environment.yml

```diff
@@ -26,19 +26,19 @@ dependencies:
 - numpydoc
 - flake8
 - spyder
+- nitime
+- nibabel
+- nilearn
```

Comment:
I would not put nitime, nibabel, pysurfer, etc. as conda packages. They are up to date on pypi but not necessarily on conda AFAIK. Please only move mayavi related stuff (traits*, pyface, etc.)